tests/lcs/core/interface/interface.livecodescript

```diff
@@ -823,4 +823,16 @@ type "a"
 
 TestAssert "test", the text of field "fi" is "bar"
 
-end TestInterface101
\ No newline at end of file
+end TestInterface101
+
+
+on TestLinkHiliteColors
+   set the linkhilitecolor to "blue"
+   TestAssert "test1", the linkhilitecolor is "blue"
+
+   set the linkhilitecolor to "255,255,0"
+   TestAssert "test2", the linkhilitecolor is "255,255,0"
+
+   set the linkhilitecolor to "#336633"
+   TestAssert "test3", the linkhilitecolor is "#336633"
+end TestLinkHiliteColors
```

Comment:
Yes sure. I think the first test should be ```TestAssert "test1", the linkhilitecolor is "255,0,0"```, since by default, the (global) linkHiliteColor is set to "255,0,0".